Dockerfile

```diff
@@ -1,33 +1,38 @@
-FROM alpine:3.4
-MAINTAINER Thomas Boerger <thomas@webhippie.de>
+#Build stage
+# TODO ARG GO_VERSION=edge
+#FROM TODO multiarch/golang:amd64-${GO_VERSION} AS build-env
+#FROM webhippie/golang:${GO_VERSION} AS build-env
+FROM sapk/alpine-multi AS build-env
+
+ARG TAGS
+ENV TAGS "bindata $TAGS"
+ENV GOPATH /go/
+ENV PATH "${PATH}:${GOPATH}/bin"
+
+RUN apk -U --no-cache add go git build-base \
+ && go get -u github.com/jteeuwen/go-bindata/...
```

Comment:
I think that it was because `git` wasn't installed so it couldn't be installed and failed after.